<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MovieDatabaseGUIClass</class>
 <widget class="QMainWindow" name="MovieDatabaseGUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>845</width>
    <height>693</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MovieDatabaseGUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListWidget" name="watchListView">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>30</y>
      <width>261</width>
      <height>481</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="addToWatchListButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>140</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>81</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Title:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Genre:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Year of release:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Likes:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Trailer:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="playWatchList">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>530</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Play</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nextWatchList">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>530</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveWatchList">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>560</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QLabel" name="updateLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>560</y>
      <width>371</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>271</width>
      <height>531</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QComboBox" name="comboBox">
       <item>
        <property name="text">
         <string>Detailed</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Short</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="repoList"/>
     </item>
     <item>
      <widget class="QLineEdit" name="titleEdit"/>
     </item>
     <item>
      <widget class="QLineEdit" name="genreEdit"/>
     </item>
     <item>
      <widget class="QLineEdit" name="yearOfReleaseEdit"/>
     </item>
     <item>
      <widget class="QLineEdit" name="likesEdit"/>
     </item>
     <item>
      <widget class="QLineEdit" name="trailerEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="addMovieButton">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteMovieButton">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="filterMoviesButton">
       <property name="text">
        <string>Filter</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="filterEdit">
       <property name="text">
        <string>Genre to filter</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="updateButton">
       <property name="text">
        <string>Update</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="oldTitleEdit">
       <property name="text">
        <string>Old title</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="removeWatchListButton">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>530</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="filenameLineEdit">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>561</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string>name.html/csv</string>
    </property>
   </widget>
   <widget class="QPushButton" name="undoButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Undo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="redoButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Redo</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>845</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MovieDatabaseGUI.qrc"/>
 </resources>
 <connections/>
</ui>
